STBReader.Tests/StbBaseTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using STBReader;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace STBReader.Tests
+{
+    [TestClass]
+    public class StbBaseTests
+    {
+        [TestMethod]
+        public void Point3Test()
+        {
+            var point3 = new Point3(3d, 3d, 3d);
+            int hashCode = point3.GetHashCode();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'hashCode'.](https://app.codacy.com/gh/hrntsm/STBReader/pullRequest?prid=6384729)